api/v1alpha1/perconaservermysqlbackup_types.go

```diff
@@ -87,3 +90,31 @@ func (cr *PerconaServerMySQLBackup) Hash() string {
 
 	return hash
 }
+
+type SidecarBackupConfig struct {
```

Comment:
Why do we now have this SidecarBackupConfig? Why sidecar backup, don't quite understand.

Reply:
We had a struct called `BackupConf` in the `cmd/sidecar.go` file. I just moved it to another package to use it in other places. I renamed it like this because we read from this struct only in the sidecar file. If you think that's not an appropriate name, we can rename it

Reply:
What do you think about just naming it like `BackupConf`, because it is just a backup conf and it is being used by the sidecar. Or maybe I'm missing something.

Reply:
I'm looking at this a bit more and if I understand it correctly this config is used only internally, not something related to users. If that is the case maybe it should not be where public API is defined. I think that is the reason why it was initially defined in the sidecar.
If you have to use it, wdyt that you put it in `xtrabackup` package. I feel it is a good fit there.

Reply:
When I placed it to `api` package, I thought that it should not be called as `BackupConf` there, because somebody could be confused with `BackupSpec`. But if we move it to the `xtrabackup` package, we can save the old name. I agree that it would be a more appropriate place for this struct. https://github.com/percona/percona-server-mysql-operator/pull/158/commits/4528a32e0c26a104b9842b349e6439428d2c7bb7